packages/tic-tac-toe/.env.development

```diff
@@ -1,7 +1,7 @@
 DEFAULT_GAS=6721975
 DEFAULT_GAS_PRICE=20000000000
 GANACHE_HOST=127.0.0.1
-GANACHE_PORT=8547
+GANACHE_PORT=8545
```

Comment:
remember that env variables are now being handled globally with the `.env.ENVIRONMENT-USE-CASE-NAME-GOES-HERE` files, and `SC_ENV` flags before the run script

Reply:
I don't really understand this. Can you explain? Either here or in Slack?